resources/data/Randomizer/doors.json

```diff
@@ -3523,13 +3523,14 @@
 		"Id": "D06Z01S08[E]",
 		"Direction": 2,
 		"OriginalDoor": "D06Z01S14[W]",
-		"Logic": "D06Z01S08[N] || D06Z01S08[E] || wallClimb"
+		"Logic": "D06Z01S08[N] || D06Z01S08[E] || wallClimb || doubleJump && canEnemyBounce"
```

Comment:
These ones that require you to repeatedly spam jump attack should probably have 'preciseSkipsAllowed' as well.  I know it is for this one, but not sure which other ones as well, can you add that where is belongs?

Reply:
Agreed. Added `preciseSkippsAllowed` to D06Z01S04[Health] and D06Z01S08[N].